### PR TITLE
feat: support 'liveBufferChasingOnPaused' config

### PR DESCRIPTION
**Pull request by FredZeng**

### CODE REVIEW

File: d.ts/mpegts.d.ts [hunk 1]
```diff
@@ -86,6 +86,13 @@ declare namespace Mpegts {
          */
         liveBufferLatencyMinRemain?: number;
 
+        /**
+         * @desc Chasing the live stream latency caused by the internal buffer in HTMLMediaElement even if HTMLMediaElement is paused
+         *       Effective only if `isLive: true` and `liveBufferLatencyChasing: true`
+         * @defaultvalue false
+         */
+        liveBufferChasingOnPaused?: boolean;
```
Comment (xqq) [inline d.ts/mpegts.d.ts:94]: It should be better to keep the naming consistent with the prefix of this series `liveBufferLatency`.
Consider renaming it into `liveBufferLatencyChasingOnPaused` and moving to the below of `liveBufferLatencyChasing`.

File: src/player/live-latency-chaser.ts [hunk 2]
```diff
@@ -57,6 +59,10 @@ class LiveLatencyChaser {
                 this._on_direct_seek(target_time);
             }
         }
+
+        if (paused && !this._media_element.paused) {
+            this._media_element.pause();
```
Comment (xqq) [inline src/player/live-latency-chaser.ts:64]: Really the assignment of `HTMLMediaElement.currentTime` triggers playback resume if the video has been paused?

Comment (FredZeng) [reply src/player/live-latency-chaser.ts:64]: not quite sure, just to be on the safe side

**Comment by xqq** [comment]
Merged, thanks!

**Pull request merged**